digital-deck-server/api/controller.js

```diff
@@ -43,7 +43,7 @@ app.post("/new-session", function (req, res) {
     isNaN(cardsOnTable)
   ) {
     throw new Error(
-      "Invalid call, needs decks, player, cardsPerPlayer and cardsOnTable as numbers in the query."
+      "Invalid call, needs decks, players, cardsPerPlayer and cardsOnTable as numbers in the query."
```

Comment:
Leaving a comment here as reminder. Since Dang added a router, this needs to be changed to returning 400 request. While merging if you have time please consider this.